demo/src/pages/interactive-examples.js

```diff
@@ -0,0 +1,280 @@
+import { React, useState } from 'react';
+import {
+  Layout,
+  CustomTextInput,
+  SignupModal,
+  Button,
+  Link,
+  SearchInput,
+} from '@newrelic/gatsby-theme-newrelic';
+import { css } from '@emotion/react';
+import PropsDisplay from '../components/PropsDisplay';
+import { graphql } from 'gatsby';
+import findComponentData from '../utils/findComponentData';
+import SelectInline from '@newrelic/gatsby-theme-newrelic/src/components/SelectInLine';
+
```

Comment:
This will be required for the other "Proptypes" code suggestion. Notably, to get this [`docgenJson`](https://github.com/newrelic/gatsby-theme-newrelic/blob/5acd30b6006f1ac8eab1aa211c409b13bb186e8e/demo/types.js) file, you might have to wait and rebase or merge this branch from `feat/update-demo` once Mary has merged her [PR](https://github.com/newrelic/gatsby-theme-newrelic/pull/806). Until then, the build might fail if you commit these suggestions.
```suggestion
import { docgenJson } from '../../types';

```

Reply:
done